build.psm1

```diff
@@ -954,7 +954,7 @@ function Install-Dotnet {
             # This is the library shipped with .NET Core
             # This is allowed to fail as the user may have installed other versions of dotnet
             Write-Warning ".NET Core links the incorrect OpenSSL, correcting .NET CLI libraries..."
-            find $env:HOME/.dotnet -name System.Security.Cryptography.Native.dylib | % { sudo install_name_tool -add_rpath /usr/local/opt/openssl/lib $_ }
+            find $env:HOME/.dotnet -name System.Security.Cryptography.Native.OpenSsl.dylib | % { sudo install_name_tool -add_rpath /usr/local/opt/openssl/lib $_ }
```

Comment:
I think you misunderstand the background. 

> Since the line is duplicated

It's not, take a closer look. One is patching in `.dotnet`, another one in `.nuget`.


There two things are independent. They been different correctly previously, it's not a problem of "changed in one place, but not in the other". Hence it's not needed to be unified.

Please, feel free to make commits to the branch, if you want to restructure it somehow.